src/firefly/js/visualize/ui/SearchSummary.jsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {flux} from '../../Firefly.js';
+import {parseWorldPt} from '../Point.js';
+import numeral from 'numeral';
+
+export class SearchSummary extends PureComponent {
+    constructor(props) {
+        super(props);
+    }
+
+    componentWillUnmount() {
+        this.iAmMounted = false;
+        if (this.removeListener) this.removeListener();
+    }
+
+    componentDidMount() {
+        this.iAmMounted = true;
+        this.removeListener = flux.addListener(() => this.storeUpdate());
+    }
+
+
+    /**
+     * If the object changed then check if any of the following changed.
+     *  This is a optimization so that the element does not re-render every time
+     */
+    storeUpdate() {
+
+    }
```

Comment:
Let's remove storeUdate, componentWillUnmount, and componentDidMount.  It's not used.

Reply:
yes.